tests/k8s_test.go

```diff
@@ -138,10 +138,11 @@ func testNodeCreation(t *testing.T, setupCmds, tearDownCmds []helper.Cmd, typ, n
 	RunTest(t, test)
 }
 
-func testNodeCreationFromConfig(t *testing.T, typ, name string, fields ...string) {
-	setup := setupFromConfigFile(typ, name)
-	tearDown := tearDownFromConfigFile(typ, name)
-	testNodeCreation(t, setup, tearDown, typ, name, fields...)
+func testNodeCreationFromConfig(t *testing.T, ty, name string, fields ...string) {
```

Comment:
Why `ty` instead of `typ` ?

Reply:
You define a `file` variable that has the same value than `ty` and you use both.

Reply:
I want to make it explicit that the type (eg pod) is also serving as file name pod(.yaml)

Reply:
As `ty` is used elsewhere in the file I wanted to make it consistent throughout